src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,127 @@
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Analytics &amp; Reporting</FooterLink>
+                                    </Grid>
+                                </Grid>
+                                <Grid item xs={6} md={4}>
+                                    <Grid container direction='row'>
+                                        <FooterLinkHeading>Help center</FooterLinkHeading>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">About</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Roadmap</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">FAQ</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Contact Us</FooterLink>
+                                    </Grid>
+                                </Grid>
+                                <Grid item xs={6} md={4} sx={isMobile ? ({ mt: 4 }) : (null)}>
+                                    <Grid container direction='row'>
+                                        <FooterLinkHeading>Legal</FooterLinkHeading>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Terms &amp; Conditions</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Pricacy Policy</FooterLink>
+                                    </Grid>
+                                </Grid>
+                            </Grid>
+                        </Box>
+                    </Grid>
+                </Grid>
+                <Grid container direction='row' sx={{ mt: 4 }}>
+                    <Grid item xs={6}>
+                        <Typography sx={{ color: '#636a7e' }}>
+                            &copy; {new Date().getFullYear()} Libernetix Inc.
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={6}>
+                        <Grid container direction='row' sx={{ justifyContent: 'flex-end' }}>
+                            <Link href='#' sx={{ mr: 2 }}>
+                                <LinkedInIcon></LinkedInIcon>
```

Comment:
Use self closing tags here and in other places there possible
`<LinkedInIcon />`